tools/mpremote/mpremote/repl.py

```diff
@@ -85,8 +102,9 @@ def console_out_write(b):
             capture_file.write(b)
             capture_file.flush()
 
+    disconnected = False
     try:
-        do_repl_main_loop(
+        disconnected = do_repl_main_loop(
```

Comment:
Can just do `return do_repl_main_loop(...)` here.  Python's finally will take care of everything else.

Reply:
Oh yes I see, that's simpler!